app/views/shared/_image_shares_modal.erb

```diff
@@ -0,0 +1,26 @@
+<div class="modal fade" id="imageSharesModal" tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel" aria-hidden="true">
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title" id="exampleModalLabel">Share an Image</h5>
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <div class="modal-body">
+        <div class ="justify-content-center">
+          <%= image_tag '', class: 'img-thumbnail' %>
+        </div>
+        <div class="container-fluid">
+          <%= simple_form_for image, remote: true, method: :post do |f| %>
+            <%= f.simple_fields_for :share do |share_form| %>
+              <%= share_form.input :email, as: :email, label: "Send to:" %>
+              <%= share_form.input :message, as: :text, label: "Message:", required: false %>
```

Comment:
Question: is this field by default `required: true`?

Reply:
Yes, simple_form assumes `required: true` by default.